### PR TITLE
Add more details on integration doc for the refusal case.

### PR DESCRIPTION
**Pull request by ahmetoz**
Closes: #641

### CODE REVIEW

File: extension/docs/WebComponentsIntegrationGuide.md [hunk 1]
```diff
@@ -728,7 +728,8 @@ Some interactions may result in a transaction. If so, the interactionId in the p
 
 ### Adyen payment refusals
 
-If you receive an HTTP 200 response with an Error or Refused resultCode, check the refusal reason and, if possible, modify your request.
+If you receive an `HTTP 200 response` with an `Error` or `Refused` resultCode from Adyen, a transaction will be added to the commercetools payment object as the refusal result of the payment request.
+Use the commercetools payment `interface interactions` to [troubleshoot](#extension-module-errors) the response.
```
Comment (butenkor) [inline extension/docs/WebComponentsIntegrationGuide.md:732]: Maybe better to link interactions with the commercetools docs? 
Linking to troubleshoot couple lines above does not seem helpful as it does not provide more infos.

File: extension/docs/WebComponentsIntegrationGuide.md [hunk 2]
```diff
@@ -728,7 +728,8 @@ Some interactions may result in a transaction. If so, the interactionId in the p
 
 ### Adyen payment refusals
 
-If you receive an HTTP 200 response with an Error or Refused resultCode, check the refusal reason and, if possible, modify your request.
+If you receive an `HTTP 200 response` with an `Error` or `Refused` resultCode from Adyen, a transaction will be added to the commercetools payment object as the refusal result of the payment request.
```
Comment (butenkor) [inline extension/docs/WebComponentsIntegrationGuide.md:731]:
```suggestion
If you receive an `HTTP 200 response` with an `Error` or `Refused` resultCode from Adyen, a transaction with a `Failure` state will be added to the commercetools payment object. Payment objects with failed transactions can not be reused for further retries. In order to retry the payment process a new commercetools payment resource needs to be created and payment steps like `makePaymentRequest` etc. re-applied.
```

File: extension/docs/WebComponentsIntegrationGuide.md [hunk 3]
```diff
@@ -737,6 +738,8 @@ Check the following table to see the mapping of Adyen [result codes](https://doc
 | Refused| Authorization (Failure)|
 | Error| Authorization (Failure)|
 
+After resolving the issue on the refused payment request, to be able to retry the request existing payment should not be reused, instead a new commercetools payment resource needs to be created and added to the appropriate cart, please refer [creating a commercetools payment](#step-2-creating-a-commercetools-payment) section for more details.
+
```
Comment (butenkor) [inline extension/docs/WebComponentsIntegrationGuide.md:742]: I guess we can skip it if we use the suggested text above.

File: extension/docs/WebComponentsIntegrationGuide.md (hunk 1 above)
Comment (ahmetoz) [reply extension/docs/WebComponentsIntegrationGuide.md]: https://github.com/commercetools/commercetools-adyen-integration/pull/646/commits/8774071da255b4dfc47f8a77d03b5381e20514fc

File: extension/docs/WebComponentsIntegrationGuide.md (hunk 2 above)
Comment (ahmetoz) [reply extension/docs/WebComponentsIntegrationGuide.md]: https://github.com/commercetools/commercetools-adyen-integration/pull/646/commits/8a1dbe2f5b2695d8b89b2c9f44e5bedbb9a2e5a1

File: extension/docs/WebComponentsIntegrationGuide.md (hunk 3 above)
Comment (ahmetoz) [reply extension/docs/WebComponentsIntegrationGuide.md]: https://github.com/commercetools/commercetools-adyen-integration/pull/646/commits/7bd98ae20f6a163a178b319106d9f84177815e9f

**Review by butenkor — APPROVED** [review]

**Pull request merged**